### DISCUSSION TITLE
pr on forked repo

### DISCUSSION THREAD

**Discussion by sujatagoudar123** [community/community#72940] [Enterprise]
### Select Topic Area

Question

### Body

Your protected branch rules won't be enforced on this private repository until you upgrade your organization to a GitHub Team or Enterprise account. even after upgrade

**Comment by Naman-B-Parlecha** [comment]
If you are seeing the message "Your protected branch rules won't be enforced on this private repository until you upgrade your organization to a GitHub Team or Enterprise account," even after upgrading your organization to a Team or Enterprise account, it may be a technical issue or delay in GitHub's systems. In such cases, I would recommend the following steps:

1. **Check Your Organization's Plan:**
   Ensure that your organization's plan has indeed been upgraded to a GitHub Team or Enterprise account. It can take some time for plan changes to propagate through GitHub's systems.

2. **Contact GitHub Support:**
   If you've confirmed the upgrade, and the issue persists, it's best to reach out to GitHub support. You can contact GitHub support through the GitHub Support portal to report the issue and get assistance with resolving it.

3. **Check Organization Settings:**
   Review your organization's settings to confirm that the appropriate permissions and access control settings are configured correctly.

GitHub support can provide you with specific guidance and assistance in resolving this issue, as they have access to your organization's account details and can investigate the problem further.

**Reply by woliarv** [reply]
We have a similar issue. We're able to enforce policies on our own repos, but not on private forks of a private repo. We can't change the pricing tier of the donor repo orga, since its not under our control, but we have access to it. Its crucial for us to make changes only regarding our orga, and have them reviewed and the branches protected, without contribution to the donor repo.  

Is there a solution for this?

**Reply by gabrielruoff** [reply]
We are having the same issue on a private fork of a private repo. Is there any solution or workaround to this besides copying over to a new repo that is owned by our org and not forked?
